ci/scripts/deterministic-e2e-test.sh

```diff
@@ -25,31 +25,30 @@ git clone https://"$GITHUB_TOKEN"@github.com/risingwavelabs/sqlsmith-query-snaps
 # popd
 popd
 
-export RUST_LOG=info
```

Comment:
Does this change make it more or less verbose?

Reply:
Aiming to decrease the test duration.

Reply:
👀 how's log level related with test duration?